suite/README.md

```diff
@@ -116,6 +116,15 @@ Inherits parameters from `install`, plus:
 
 `replace_variety` will generate a combination of `replace` parameterized tests.
 
+### Post installer transfer script
```

Comment:
Post bootstrap?

Reply:
Maybe
```
### Post Installer Bootstrap Script
```

Reply:
no it's specifically post installer transfer.